### PR TITLE
Revert "Smc29 kildeer"

### PR DESCRIPTION
**Pull request by smc29**
Reverts githubteacher/everyone-may-2016#233

### CODE REVIEW

File: smc29-kildeer.md [hunk 1]
```diff
@@ -1,10 +0,0 @@
-# Kildeer, IL
-
-### Places to Eat
- - Every chain restaurant imaginable
- 
-### Touristy Things to Do
- - Visit Chicago (35 Mi SW)
- 
-### Secret Spots for Locals
- - The Chatterbox in nearby Long Grove
```
Comment (FherGlez) [inline smc29-kildeer.md]: Please more info

**Comment by ryanmorillo** [comment]
Why revert? Nothing going on here.

**Comment by smc29** [comment]
There isn't much more to add - it's a tiny bedroom suburb of Chicago - <3500 people!

**Comment by smc29** [comment]
I reverted because I merged too soon - at least I thought it did...this is kind of fast and I got interrupted!

**Pull request merged**